lib/CrudService.js

```diff
@@ -86,7 +87,7 @@ class CrudService {
   }
 
   findAll(context, query, projection, sort, skip, limit, _states, isTextSearchQuery) {
-    const allowedStates = _states || [PUBLIC]
+    const allowedStates = _states ?? [PUBLIC]
```

Comment:
Why this function has a different behavior from all the others?

Reply:
My bad, I'm changing